Controllers/LoginController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using managementapp.Data;
+using managementapp.Data.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace managementapp.Controllers
+{
+    public class LoginController : Controller
+    {
+        private readonly ITokenService _tokenService;
+        private readonly DataContext _context;
+
+
+        public LoginController(DataContext context, ITokenService tokenService)
+        {
+            _context = context;
+            _tokenService = tokenService;
+        }
+        [HttpPost("Login")]
+        public async Task<ActionResult<List<UserLogin>>> Login(DTOLogin request)
+        {
+            var user =  await _context.UserLogins.SingleOrDefaultAsync(u => u.Email==request.Email);
```

Comment:
public UserModel Authenticate(UserDTO user)
{
    var currentUser = _userRepository.GetByUsernameAndPassword(user.Email, user.Password); // Use the UserRepository to get the user by username and password
    if (currentUser != null)
    {
        return currentUser;
    }
    return null;
}

thử lại cách này nha nó phải check đúng pass với mail mới ok